app/views/branded/plans/_navigation.html.erb

```diff
@@ -28,7 +28,7 @@ end
       </a>
     </li>
   <% end %>
-  <% if Rails.configuration.x.enable_research_outputs && plan.template.enable_research_outputs? %>
+  <% if Rails.configuration.x.enable_research_outputs && (plan.template.enable_research_outputs? || plan.research_outputs.length > 0) %>
```

Comment:
change `plan.research_outputs.length > 0` to `plan.research_outputs.any?`

Reply:
https://github.com/CDLUC3/dmptool/commit/647d5b783a576816c399d7e83da52721f928c205